ansible/roles/contiv_storage/defaults/main.yml

```diff
@@ -3,7 +3,7 @@
 # Role defaults for contiv_storage
 
 contiv_storage_supervisor_host: ""
-contiv_storage_version: "v0.1.0"
+contiv_storage_version: "v0.0.0-06-30-2016.09-52-49.UTC"
```

Comment:
I just removed these releases. Please use the official releases. v0.2 should be out tonight.